pkg/client/group_mgmt_client.go

```diff
@@ -20,7 +20,7 @@ const (
 	clientTimeout     = time.Minute
 	maxLoginRetries   = 1
 	jobTimeout        = time.Minute * 5
```

Comment:
I was going to let it go, but it is still bugging me...  Can we make it "5 * time.Minute" to be consistent with the line below?  Also remove the useless comment on the line below.